content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Introducing the Istio v1beta1 Authorization Policy
+description: Introduction, motivation and design principles for the Istio v1beta1 Authorization Policy.
+publishdate: 2019-11-11
+subtitle:
+attribution: Yangmin Zhu (Google)
+keywords: [security, RBAC, access control, authorization]
+target_release: 1.4
+---
+
+Up until now, Istio has provided the role based access control (RBAC) policy for enforcing access control
+on services using three configuration resources: `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`.
+With this API, users have been able to enforce access control in mesh-level, namespace-level and
+service-level. Like any other RBAC policy, Istio RBAC uses the same concept of role and binding for granting
+permissions to identities.
+
+While this functionality has proven to be working for many customers, we have also recognised
+some shortcomings in its design that is causing many confusions and preventing its further adoption.
```

Comment:
I would not call it "shortcoming in its design". I would say, v1beta1 policy improves the alpha policy in several ways.
1. replace service model with workload model for better clarity on the policy target. 
2. Simplification on the policy syntax. 
3. Allow the policies to be applied on Ingress/Egress gateways.

Reply:
Removed the shortcoming part, Also updated the `design principles` to integrate the 3 points here.